roles/acme/tasks/challenge/dns-01/hetzner.yml

```diff
@@ -8,7 +8,7 @@
         url: https://dns.hetzner.com/api/v1/zones?name={{ acme_domain.zone }}
         headers:
           Auth-API-Token: "{{ acme_hetzner_auth_token }}"
-      register: lookup_zone_id
+      register: acme_lookup_zone_id_result
```

Comment:
this seems unused?

Reply:
fixed. Was used later but the used name was also the old on. Thank you! :)